### ISSUE TITLE
Handle response from slack better

### ISSUE THREAD

**Issue by parberge** [codelabsab/timereport-slack#67]
Right now we only check if slack responded http a 200 or not.

What we should do is to check the response text for this:
```
timereport - DEBUG - Slack client response was:
{
    "ok": true,
    "message_ts": "1550913733.049000"
}
```
And take appropiate action if "ok" != true

**Comment by kamigerami** [comment]
Ok.

**Comment by parberge** [comment]
Fixed in release `1.10.3`
